e2e-tests/functions

```diff
@@ -365,6 +365,42 @@ deploy_pmm_server() {
 	fi
 }
 
+deploy_pmm3_server() {
+	helm uninstall -n "${NAMESPACE}" pmm || :
+	if [[ $OPENSHIFT ]]; then
+		platform=openshift
+		oc create sa pmm-server -n "$NAMESPACE"
+		oc adm policy add-scc-to-user privileged -z pmm-server -n "$NAMESPACE"
+
+		if [[ $OPERATOR_NS ]]; then
+			timeout 30 oc delete clusterrolebinding $(kubectl get clusterrolebinding | grep 'pmm-pg-operator-' | awk '{print $1}') || :
+			oc create clusterrolebinding pmm-pg-operator-cluster-wide --clusterrole=percona-postgresql-operator --serviceaccount=$NAMESPACE:pmm-server -n "$NAMESPACE"
+			oc patch clusterrole/percona-postgresql-operator --type json -p='[{"op":"add","path": "/rules/-","value":{"apiGroups":["security.openshift.io"],"resources":["securitycontextconstraints"],"verbs":["use"],"resourceNames":["privileged"]}}]' ${OPERATOR_NS:+-n $OPERATOR_NS}
+		else
+			oc create rolebinding pmm-pg-operator-namespace-only --role percona-postgresql-operator --serviceaccount=$NAMESPACE:pmm-server -n "${NAMESPACE}"
+			oc patch role/percona-postgresql-operator --type json -p='[{"op":"add","path": "/rules/-","value":{"apiGroups":["security.openshift.io"],"resources":["securitycontextconstraints"],"verbs":["use"],"resourceNames":["privileged"]}}]' -n "$NAMESPACE"
+		fi
+		helm install monitoring --set imageTag=${IMAGE_PMM3_SERVER#*:} --set imageRepo=${IMAGE_PMM3_SERVER%:*} --set platform=$platform --set sa=pmm-server --set supresshttp2=false https://percona-charts.storage.googleapis.com/pmm-server-${PMM3_SERVER_VERSION}.tgz -n "$NAMESPACE"
+	else
+		platform=kubernetes
+
+		helm uninstall -n "${NAMESPACE}" monitoring || :
+		helm repo remove percona || :
+		kubectl delete clusterrole monitoring --ignore-not-found
+		kubectl delete clusterrolebinding monitoring --ignore-not-found
+
+		helm repo add percona https://percona.github.io/percona-helm-charts/
+		helm install monitoring percona/pmm -n "${NAMESPACE}" \
+			--set fullnameOverride=monitoring \
+			--version 1.4.0 \
+			--set imageTag=3-dev-latest \
+			--set imageRepo=perconalab/pmm-server \
```

Comment:
Versions here are hardcoded because we still have the `monitoring` test and it will run together with the `monitoring-pmm3` test for some time. The `monitoring` test is configured through env vars or default values and introducing additional env vars for the pmm3 can lead to unnecessary redundancy. When PMM3 is the only option for running PMM with the PG operator probably we can switch to env vars this setup here. WDYT?

Reply:
If we remove completely the version then, it will always default to the latest one 

```
		helm install monitoring percona/pmm  \
			--set fullnameOverride=monitoring \
			--set imageTag=3-dev-latest \
			--set imageRepo=perconalab/pmm-server 
```